db/config.cc

```diff
@@ -1325,7 +1325,8 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , minimum_replication_factor_warn_threshold(this, "minimum_replication_factor_warn_threshold", liveness::LiveUpdate, value_status::Used,  3, "")
     , maximum_replication_factor_warn_threshold(this, "maximum_replication_factor_warn_threshold", liveness::LiveUpdate, value_status::Used, -1, "")
     , maximum_replication_factor_fail_threshold(this, "maximum_replication_factor_fail_threshold", liveness::LiveUpdate, value_status::Used, -1, "")
-    , tablets_initial_scale_factor(this, "tablets_initial_scale_factor", value_status::Used, 10, "Calculated initial tablets are multiplied by this number")
+    , tablets_initial_scale_factor(this, "tablets_initial_scale_factor", value_status::Used, 10,
+         "Minimum average number of tablet replicas per shard per table. Suppressed by tablet hints in table's schema: min_per_shard_tablet_count and min_tablet_count")
```

Comment:
nit: rename `hints` to `options`
Also, how about the expected_table_size_in_gb option?  Maybe we should list the particular option and leave this open to any per-table option may take priority over the configuration option, which can be described as just the default.

Reply:
nit in commit `tablets, topology: Introduce tablets_per_shard_goal config` log message: s/decrese/decrease/

Reply:
> nit: rename `hints` to `options` 

Yes

> Also, how about the expected_table_size_in_gb option? Maybe we should list the particular option and leave this open to any per-table option may take priority over the configuration option, which can be described as just the default.

expected_table_size_in_gb doesn't suppress it, because it doesn't lower the minimum. It would suppress it if the count could go below that which follows from `tablets_initial_scale_factor` logic. But the meaning of the option is to set a minimum, not a particular count, so if the count goes higher, the option is still in effect.